site/build.gradle.kts

```diff
@@ -45,6 +45,7 @@ kobweb {
             ".components.widgets.filesystem.Folders",
             ".components.widgets.navigation.DocsLink",
             ".components.widgets.navigation.DocsAside",
+            ".components.widgets.video.YoutubeVideo"
```

Comment:
We don't need this to be global. Instead, in `Videos.md`, add to the front matter:

```
---
...
imports:
  - .components.widgets.video.YouTubeVideo
---
```